plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergGetTableStatisticsOperations.java

```diff
@@ -92,6 +92,9 @@ public void tearDown()
             throws IOException
     {
         deleteRecursively(metastoreDir.toPath(), ALLOW_INSECURE);
+        localQueryRunner.close();
```

Comment:
closer? can `deleteRecursively` fail. And even if not we should null fields on failure in `localQueryRunner.close`

Reply:
It doesn't have to be super-resilient. Cleanups generally don't fail and if they do, we know about that ( see comment above https://github.com/trinodb/trino/pull/15165#discussion_r1033532569).

I optimize for readability here.